src/test/java/com/iota/iri/TransactionTestUtils.java

```diff
@@ -213,6 +232,23 @@ public static Transaction get9Transaction() {
 
         return buildTransaction(trits);
```

Comment:
Implicit casting! `buildTransaction` returns Transaction while this method `byte[]`. Typing should be fixed here and other occurrences. I know you did not write this line but it should be fixed.

Reply:
Im not sure what you mean here, maybe its from old code?